### PR TITLE
Improves model builded by Java reflection

### PR DESCRIPTION
**Pull request by GerardPaligot**

### CODE REVIEW

File: src/main/java/spoon/support/visitor/java/reflect/RtMethod.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+	private Class<?> returnType;
+	private TypeVariable<Method>[] typeParameters;
+	private Class<?>[] parameterTypes;
+	private Class<?>[] exceptionTypes;
+	private int modifiers;
+	private Annotation[] annotations;
+	private Annotation[][] parameterAnnotations;
+	private boolean isVarArgs;
+
+	public RtMethod(Class<?> clazz, String name, Class<?> returnType, TypeVariable<Method>[] typeParameters, Class<?>[] parameterTypes, Class<?>[] exceptionTypes, int modifiers, Annotation[] annotations,
```
Comment (tdurieux) [inline src/main/java/spoon/support/visitor/java/reflect/RtMethod.java]: java.lang.reflect.Method is final?

Comment (GerardPaligot) [inline src/main/java/spoon/support/visitor/java/reflect/RtMethod.java]: Yes

**Comment by monperrus** [comment]
fine for me. merge?

**Comment by tdurieux** [comment]
> Results :

> Failed tests:   testDeclaredField(spoon.support.visitor.java.JavaReflectionTreeBuilderTest): expected:<3> but was:<4>
 
In travis, I don t know why?

**Comment by GerardPaligot** [comment]
@tdurieux Travis seems ok. Where do you find this?

**Comment by tdurieux** [comment]
at the end of the raw log

**Comment by GerardPaligot** [comment]
So weird. I'll check that.

**Comment by GerardPaligot** [comment]
Nice, a field is added when we apply the coverage. :)

**Comment by monperrus** [comment]
Cool explanation. And why does Travis remains green?

**Comment by GerardPaligot** [comment]
The coverage never fails.

**Pull request merged**